### PR TITLE
Update default docker images

### PR DESCRIPTION
**Pull request by hzy46**
This PR builds new docker images to change the previous ufoym/deepo images. The target is to support SSH for our default docker images

The building process uses the following Dockerfile:

```
FROM <BASE>
RUN apt-get update
RUN apt-get -y install openssh-server curl
```

### CODE REVIEW

**Comment by hzy46** [comment]
> I did not check whether these new dockers work or not, please make sure them working end to end. thanks for updating the images with needed libraries, very useful for the customers.

I have checked all the images by simple examples. All of them work well and have SSH info.

File: docs/user/job_submission.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ The Hello World job is set to download the CIFAR-10 dataset and train a simple m
 
   OpenPAI uses [Docker](https://www.docker.com/why-docker) to provide consistent and independent environments. With Docker, OpenPAI can serve multiple job requests on the same server. The job environment depends significantly on the docker image you select.
 
-  The hub.docker.com is a public Docker repository. And the [ufoym/deepo](https://hub.docker.com/r/ufoym/deepo) on hub.docker.com is recommended for deep learning. In the hello-world example, it uses a TensorFlow image, *ufoym/deepo:tensorflow-py36-cu90*, in ufoym/deepo. You can also set your own image from private repository by toggling custom button.
```
Comment (debuggy) [inline docs/user/job_submission.md]: add openpai？

File: docs/user/job_submission.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ The following process submits a model training job implemented by TensorFlow on
 
 4. Specify the resources you need. By default only gpu number could be set. Toggle the "custom" button if you need to customize CPU number and memory. Here we use the default setting which utilizes one GPU.
 
-5. Specify the docker image. You can either use the listed docker images or take advantage of your own one. Here we use "ufoym/deepo:tensorflow-py36-cu90" as the docker image. OpenPAI will pull images from the [official Docker Hub](https://hub.docker.com/). If you want to use your own Docker registry, please click the "Auth" button and fill in the required information.
```
Comment (debuggy) [inline docs/user/job_submission.md]: add openpai?

**Comment by debuggy** [comment]
have you tested if the openpai image worked as expected

**Comment by ShengleiH** [comment]
@hzy46 Hi, I am also trying to connect by SSH but I failed. What I did is as follows:
1. Edit commands in "submit job" page:
```
echo "FROM <BASE>\nRUN apt-get update\nRUN apt-get -y install openssh-server curl" | sed 's/\\n/\n/g' > Dockerfile
docker build -f Dockerfile .
sleep infinity
```
2. I choose "ufoym/deepo:tensorflow-py36-cu90" docker image

But the result is :
```
[Exit Trigger Info]

ExitTriggerMessage: FailedTaskCount 1 has reached MinFailedTaskCount 1 in TaskRole [Task_role_1]
ExitTriggerTaskRole: Task_role_1
ExitTriggerTaskIndex: 0
UserExitCode: 127

--------------------------------------------------------------------------------

[Exit Spec]

code: 255
phrase: CONTAINER_RUNTIME_UNKNOWN_FAILURE
issuer: PAI_RUNTIME
causer: UNKNOWN
type: UNKNOWN_FAILURE
stage: COMPLETING
behavior: UNKNOWN
reaction: RETRY_TO_MAX
reason: Container failed but the failure cannot be recognized by PAI Runtime
repro:
  - User program directly exits with exitcode 1
solution:
  - Check container log and find root cause
  - Wait result from next retry


--------------------------------------------------------------------------------

[Exit Diagnostics]

[2019-08-19 09:07:36.029]Exception from container-launch: 
ExitCodeException exitCode=255: 
	at org.apache.hadoop.util.Shell.runCommand(Shell.java:998)
	at org.apache.hadoop.util.Shell.run(Shell.java:884)
	at org.apache.hadoop.util.Shell$ShellCommandExecutor.execute(Shell.java:1216)
	at org.apache.hadoop.yarn.server.nodemanager.DockerContainerExecutor.launchContainer(DockerContainerExecutor.java:312)
	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.launchContainer(ContainerLaunch.java:443)
	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.call(ContainerLaunch.java:294)
	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.call(ContainerLaunch.java:92)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

[2019-08-19 09:07:36.029]
[2019-08-19 09:07:36.030]Container exited with a non-zero exit code 255. Last 40960 bytes of runtime.pai.agg.error :
[PAI_RUNTIME_ERROR_START]
exitcode: 255
trigger: originalUserExitCode
reason: |
  User process exited with code: 127
solution: |
  
originalUserExitCode: 127
dockerStatus: 
caughtSignal: 
caughtException:
matchedUserLogString:
errorLogs:
  user: |
    /pai/bootstrap/docker_bootstrap.sh: line 206: docker: command not found
  runtime: |
    1566205654 ERROR REASON "User process exited with code: 127"
    WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
    1566205654 INFO USER_EXIT_CODE 127
  diskCleaner: |
[PAI_RUNTIME_ERROR_END]

[2019-08-19 09:07:36.031]
```

Did I misunderstand something? Thank you!

**Comment by hzy46** [comment]
> @hzy46 Hi, I am also trying to connect by SSH but I failed. What I did is as follows:
> 
> 1. Edit commands in "submit job" page:
> 
> ```
> echo "FROM <BASE>\nRUN apt-get update\nRUN apt-get -y install openssh-server curl" | sed 's/\\n/\n/g' > Dockerfile
> docker build -f Dockerfile .
> sleep infinity
> ```
> 
> 1. I choose "ufoym/deepo:tensorflow-py36-cu90" docker image
> 
> But the result is :
> 
> ```
> [Exit Trigger Info]
> 
> ExitTriggerMessage: FailedTaskCount 1 has reached MinFailedTaskCount 1 in TaskRole [Task_role_1]
> ExitTriggerTaskRole: Task_role_1
> ExitTriggerTaskIndex: 0
> UserExitCode: 127
> 
> --------------------------------------------------------------------------------
> 
> [Exit Spec]
> 
> code: 255
> phrase: CONTAINER_RUNTIME_UNKNOWN_FAILURE
> issuer: PAI_RUNTIME
> causer: UNKNOWN
> type: UNKNOWN_FAILURE
> stage: COMPLETING
> behavior: UNKNOWN
> reaction: RETRY_TO_MAX
> reason: Container failed but the failure cannot be recognized by PAI Runtime
> repro:
>   - User program directly exits with exitcode 1
> solution:
>   - Check container log and find root cause
>   - Wait result from next retry
> 
> 
> --------------------------------------------------------------------------------
> 
> [Exit Diagnostics]
> 
> [2019-08-19 09:07:36.029]Exception from container-launch: 
> ExitCodeException exitCode=255: 
> 	at org.apache.hadoop.util.Shell.runCommand(Shell.java:998)
> 	at org.apache.hadoop.util.Shell.run(Shell.java:884)
> 	at org.apache.hadoop.util.Shell$ShellCommandExecutor.execute(Shell.java:1216)
> 	at org.apache.hadoop.yarn.server.nodemanager.DockerContainerExecutor.launchContainer(DockerContainerExecutor.java:312)
> 	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.launchContainer(ContainerLaunch.java:443)
> 	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.call(ContainerLaunch.java:294)
> 	at org.apache.hadoop.yarn.server.nodemanager.containermanager.launcher.ContainerLaunch.call(ContainerLaunch.java:92)
> 	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
> 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
> 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
> 	at java.lang.Thread.run(Thread.java:748)
> 
> [2019-08-19 09:07:36.029]
> [2019-08-19 09:07:36.030]Container exited with a non-zero exit code 255. Last 40960 bytes of runtime.pai.agg.error :
> [PAI_RUNTIME_ERROR_START]
> exitcode: 255
> trigger: originalUserExitCode
> reason: |
>   User process exited with code: 127
> solution: |
>   
> originalUserExitCode: 127
> dockerStatus: 
> caughtSignal: 
> caughtException:
> matchedUserLogString:
> errorLogs:
>   user: |
>     /pai/bootstrap/docker_bootstrap.sh: line 206: docker: command not found
>   runtime: |
>     1566205654 ERROR REASON "User process exited with code: 127"
>     WARNING: Your kernel does not support swap limit capabilities or the cgroup is not mounted. Memory limited without swap.
>     1566205654 INFO USER_EXIT_CODE 127
>   diskCleaner: |
> [PAI_RUNTIME_ERROR_END]
> 
> [2019-08-19 09:07:36.031]
> ```
> 
> Did I misunderstand something? Thank you!

Hi, ShengleiH, you should not add the "docker build" commands in your job config. These commands are used to ***build docker images***. You should build the images locally and push it to Docker hub. Then use that image in your job config. More information: https://github.com/microsoft/pai/blob/master/docs/job_docker_env.md#enable-ssh-for-your-image

**Pull request merged**